ExchangeWebServices/Source/MS-OXWSCORE/TestSuite/MS-OXWSCORE_RequirementSpecification.xml

```diff
@@ -6320,15 +6336,16 @@
 	<ns1:Requirement>
 		<ns1:REQ_ID>MS-OXWSCORE_R2142</ns1:REQ_ID>
 		<ns1:Doc_Sect>2.2.4.43</ns1:Doc_Sect>

```

Comment:
It should be derived RS to verify like MS-OXWSCORE_R1730. Please add New RS and Comments.

Reply:
Adapter